students/minha/westagram/users/views.py

```diff
@@ -48,13 +49,14 @@ def post(self, request):
 
             if User.objects.filter(email=email).exists():
                 return JsonResponse({'message':'DUPLICATED_EMAIL'}, status=400) #서로 다른 사람이 같은 이메일 사용 불가
-
+            
+            hashed_password  = bcrypt.hashpw( password.encode('utf-8'), bcrypt.gensalt() )
             User.objects.create(
                 username     = username,
                 first_name   = first_name,
                 last_name    = last_name,
                 email        = email,
-                password     = password,
+                password     = hashed_password,
```

Comment:
bcrypt로 hash한 값의 데이터 타입은 byte입니다. byte 타입을 그대로 db에 저장하면 문제가 생길 수 있습니다. 따라서 해당 결과를 다시 str로 decode한 후에 db에 저장해야 합니다.

```suggestion
                password     = hashed_password.decode('utf-8'),
```